dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/JdbcSelectTask.java

```diff
@@ -35,6 +37,10 @@ public class JdbcSelectTask extends AbstractJdbcTask<Summary> {
 
   @Nonnull private final TaskCategory taskCategory;
 
+  private boolean calculateQueryLogDates;
```

Comment:
Introduced this to differentiate cases when we don't need consider dates as first or last query log entry.
For instance in SnowflakeLogsConnector there are time series tasks, which run by 1 day interval, so dates are coming here and calculating as first entry and last entry, which is incorrect.